recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "0.15.1" %}
+{% set version = "2019.7.26" %}
```

Comment:
Is this older or newer than 0.15.1 :smile:

Reply:
@ocefpaf it appears the versioning scheme changed between `0.15.1` and `2018.10.18`